pages/user guides/importing_from_excel.md

```diff
@@ -7,9 +7,11 @@ sidebar: main_sidebar
 permalink: import-from-excel.html
 ---
 
-Tidal's importer will guide you through mapping your columns to our fields,
-create your own fields and even make associations between dependencies
-if you have captured these.
+Tidal's importer will guide you through mapping your columns to our
+fields, creating your own fields, and even making associations between
+dependencies if you have captured these. When using the Tidal
+importer, we will create or update records depending on the name's or
```

Comment:
I find the first-person presentation strange myself as if the software is a 'we'. I don't see it used elsewhere on the guide. _it will create_, seems more natural, to me at least.

I think it's not possessive (or contraction) and only plural here.
```suggestion
importer, we will create or update records depending on the names or
```

Reply:
That is what grammarly told me to do, but maybe it got confused

Reply:
oh ok gotcha! I very well could have been confused myself on that. I'd trust grammarly interpretation